frontend/src/Pages/DishesSelectedPage.js

```diff
@@ -1,25 +1,35 @@
 import PropTypes from 'prop-types'
 import styled from 'styled-components/macro'
 import HeaderOverlay from '../Components/HeaderOverlay/HeaderOverlay'
-import ListItem from '../Components/ListItem/ListItem'
+import DishListItem from '../Components/DishListItem/DishListItem'
 
 SelectedDishesPage.propTypes = {
   dishes: PropTypes.array,
   onDeleteItem: PropTypes.func,
 }
```

Comment:
are they required? in this case you can use 
```suggestion
onDeleteItem: PropTypes.func.isRequired
```
maybe better for error-handling